docs/ldml/tr35-numbers.html

```diff
@@ -2539,7 +2539,7 @@ <h3>5.1 <a name="Plural_rules_syntax" href=
 sampleList    = sampleRange (',' sampleRange)* (',' ('…'|'...'))?
 sampleRange   = decimalValue ('~' decimalValue)?
 value         = digit+
-decimalValue  = value ('.' value)?<br>digit         = 0|1|2|3|4|5|6|7|8|9
+decimalValue  = value ('.' value)?<span class="changed"> ('e' value)?</span><br>digit         = 0|1|2|3|4|5|6|7|8|9
```

Comment:
Consider forbidding `e0`, since I believe `1e0` is equivalent to `1` in the eyes of the spec.  In compact decimals in English, for example, "123" has e = 0, but "1.2K" has e = 3.

Reply:
Fixed.